### PR TITLE
MES-8242 / Candidate licence details

### PR DESCRIPTION
**Pull request by matthew2564**
## Description
- Candidate details view
## Checklist

- [ ] PR title includes the JIRA ticket number
- [ ] Branch is rebased against the latest develop
- [ ] Code has been tested manually
- [ ] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: src/app/pages/candidate-licence/candidate-licence.actions.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+export const GetCandidateLicenceDataSuccess = createAction(
+  '[Candidate licence] Get licence data success',
+);
+
+export const GetCandidateLicenceDataError = createAction(
+  '[Candidate licence] Get licence data error',
+);
+
+// @TODO: Dispatch action.
```
Comment (matthew2564) [inline src/app/pages/candidate-licence/candidate-licence.actions.ts]: Need to confirm with team.

File: src/app/pages/candidate-licence/candidate-licence.effects.ts [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    catchError((err) => {
+      this.store$.dispatch(SaveLog({
+        payload: this.logHelper.createLog(LogType.ERROR, 'Error retrieving candidate licence', err.error),
+      }));
+      return of(GetCandidateLicenceDataError());
+    }),
+    repeat(),
+  ));
+
+  // @TODO: Decide when to dump data
```
Comment (matthew2564) [inline src/app/pages/candidate-licence/candidate-licence.effects.ts]: Need to confirm with team.

File: src/app/pages/test-report/cat-adi-part3/components/adi3-end-test-modal/adi3-end-test-modal.html [hunk 3]
```diff
@@ -16,7 +16,7 @@
   </modal-result-item>
```
Comment (matthew2564) [inline src/app/pages/test-report/cat-adi-part3/components/adi3-end-test-modal/adi3-end-test-modal.html]: Undo file. (*was helping cai with automation stuff).

File: src/app/pages/candidate-licence/__tests__/candidate-licence.page.spec.ts [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+  CandidateLicenceDataValidationError,
+  CandidateLicenceViewDidEnter,
+} from '@pages/candidate-licence/candidate-licence.actions';
+import { TrueLikenessToPhotoChanged } from '@store/tests/test-summary/test-summary.actions';
+import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+import { Store } from '@ngrx/store';
+import { StoreModel } from '@shared/models/store.model';
+import { TestFlowPageNames } from '@pages/page-names.constants';
+
+fdescribe('CandidateLicencePage', () => {
```
Comment (andrewsetterfield) [inline src/app/pages/candidate-licence/__tests__/candidate-licence.page.spec.ts]: f

Comment (matthew2564) [reply src/app/pages/candidate-licence/__tests__/candidate-licence.page.spec.ts]: Yes, did say please review whilst i finish tests 👍  (so ignore any .specs)

**Review by andrewsetterfield — APPROVED** [review]

**Review by RLCorp — APPROVED** [review]

**Pull request merged**